connection.go

```diff
@@ -479,13 +488,15 @@ func (c *Connection) readLoop() {
 	for {
 		messageLength, err = c.readMessageLength(r)
 		if err != nil {
+			c.handleError(fmt.Errorf("reading message length: %w", err))
```

Comment:
I know moovfinancial/errors isn't open source, but do we want to wrap these errors? I believe pkg/errors is compatible with our closed source errors package.

Reply:
Do you see an issue with exposing sensitive data, or it's something else?

Reply:
I like the SafeError proposal instead.